test/bigtest/tests/units/settings/MappingProfilesForm/TransformationsModal-test.js

```diff
@@ -42,6 +42,10 @@ const initialValues = {
   ],
 };
 
+const totalSelectedMessage = selectedCount => {
```

Comment:
Could you please write the function (or create a new) in the way to make it usable also for the fields found case?
https://github.com/folio-org/ui-data-export/pull/122/files#diff-491c106836b32bfb2aaaf5a44c7de2f4R283